### PR TITLE
Remove strange third-party links that mirror old MDN pages

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes a bunch of strange third-party links for `contest-server.cs.uchicago.edu` that mirror old MDN articles.  It seems that this domain has entire copies of language references, including one for JavaScript which is a copy of pre-Yari MDN.  It seems strange that we'd be linking to a mirror of our own pages, even though the pages linked are long gone from MDN.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
The intent _is_ to link to a deleted page. Maybe link to the Internet archive if this link does not look secure enough?

**Comment by queengooborg** [comment]
I see that these links came from #16772.  In that PR, however, some links are redirected to this archive URL and some are removed -- and given how we usually handle irrelevant technologies, I would have assumed they'd all be removed.

@OnkarRuikar, would you mind providing some additional context for these links?

File: files/en-us/mozilla/firefox/releases/19/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Support for [XForms](/en-US/docs/XForms) has been [**removed**](https://www.phil
...
 
 - `getBrowserSelection()` now returns the selected text in a text input field. As a result, `gContextMenu.isTextSelected` will be `true` when the user selects text in a text input field that is not a password field. ([Firefox bug 565717](https://bugzil.la/565717))
-- [Dict.jsm](/en-US/docs/Mozilla/JavaScript_code_modules/Dict.jsm): [`Dict()`](/en-US/docs/Mozilla/JavaScript_code_modules/Dict.jsm#creating_a_dictionary) now takes a JSON String. [`Dict.toJSON()`](<https://contest-server.cs.uchicago.edu/ref/JavaScript/developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/Dict.html#toJSON()>) was added, and it returns a JSON String. ([Firefox bug 727967](https://bugzil.la/727967))
+- [Dict.jsm](/en-US/docs/Mozilla/JavaScript_code_modules/Dict.jsm): [`Dict()`](/en-US/docs/Mozilla/JavaScript_code_modules/Dict.jsm#creating_a_dictionary) now takes a JSON String. `Dict.toJSON()` was added, and it returns a JSON String. ([Firefox bug 727967](https://bugzil.la/727967))
```
Comment (OnkarRuikar) [inline files/en-us/mozilla/firefox/releases/19/index.md:50]: https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/Dict.jsm
These links do not exist. Why are we replacing working links with non working ones? :confused :

Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/19/index.md:50]:
```suggestion
- Dict.jsm]: `Dict()` now takes a JSON String. `Dict.toJSON()` was added, and it returns a JSON String. ([Firefox bug 727967](https://bugzil.la/727967))
```

Comment (queengooborg) [reply files/en-us/mozilla/firefox/releases/19/index.md:50]: We're not; that link was already there.  We're removing the links that point to an old MDN mirror (contest-server.cs.uchicago.edu).

**Comment by OnkarRuikar** [comment]
> @OnkarRuikar, would you mind providing some additional context for these links?

At that time these were the only available reliable references to these APIs. Though the APIs no longer exist I was instructed to keep the references to preserve the historical info. As these are old release notes, no one is ever going to really try to use them.

**Review by hamishwillee — APPROVED** [review]
Thanks @queengooborg - good fixes (modulo a little tidy up).

Periodically pages get deleted and people try to find archives to link to the original information. In Firefox release notes (at least) it is almost always better to just delete the links as done here. 
No one is worse off, and creating a link means potential for ongoing maintenance in order to support something that has no relevance.

**Comment by hamishwillee** [comment]
PS @OnkarRuikar thanks for your diligence. I don't see value in the historical info vs the cost of maintenance. If someone strongly disagrees they can restore from github - but I will oppose such a PR.

**Pull request merged**